usr/share/rear/lib/global-functions.sh

```diff
@@ -534,7 +534,7 @@ function mount_url() {
                     wilful_input=""
                     symlink_target=""
                     # When USER_INPUT_RELAXRECOVER_SYMLINK_TARGET has any 'true' value be liberal in what you accept and
-                    # assume choices[0] 'Let /dev/disk/by-label/RELAXRECOVER point to /dev/cdrom' was actually meant:
+                    # assume choices[0] 'Let /dev/disk/by-label/REAR-ISO point to /dev/cdrom' was actually meant:
                     is_true "$USER_INPUT_RELAXRECOVER_SYMLINK_TARGET" && USER_INPUT_RELAXRECOVER_SYMLINK_TARGET="${choices[0]}"
```

Comment:
Please also change the variable name
```
USER_INPUT_RELAXRECOVER_SYMLINK_TARGET
```
that is and was wrong from the beginning
(because the hardcoded name contains a user config value)
to
```
USER_INPUT_ISO_VOLID_SYMLINK_TARGET
```
which is a name that matches its meaning.

I know changing this variable name is not backward compatible
because some users might already use that variable name.
But I assume nobody (or at least almost) uses it in practice.
I think no harm happens when the old name does no longer work.
Only an automated user input would no longer work so
only in case of unattended recovery the behaviour changes.
I will document this minor backward incompatible change
in the release notes for ReaR 2.7.

Reply:
@pcahyna 
if you do not like to change the variable name
via this pull request I would do it in a
subsequent separated pull request.

Reply:
I prefer to do the variable name change in a separate pull request from the innocuous comment update.

Reply:
Via https://github.com/rear/rear/pull/2814
I renamed the user config variable for automated input
USER_INPUT_RELAXRECOVER_SYMLINK_TARGET
as USER_INPUT_ISO_SYMLINK_TARGET

Reply:
I mean, you wrote that you would document it, but where are the entries for release notes kept?

Reply:
The current release notes sources are in
https://github.com/rear/rear.github.com/tree/master
therein
https://github.com/rear/rear.github.com/blob/master/documentation/release-notes-2-6.md

The work-in-progress release notes for ReaR 2.7 are
currently in this branch
https://github.com/rear/rear.github.com/tree/jsmeix-release-notes-2-7
therein
https://github.com/rear/rear.github.com/blob/jsmeix-release-notes-2-7/documentation/release-notes-2-7.md

There I will add the recent changes
including this one next week.

Reply:
Thanks. I have an idea, what about using a special label on PRs indicating "this PR requires a release notes entry"?

Reply:
@pcahyna @jsmeix you could also use a small prep script to copy old variable names into the new ones. We did this in the past for NETFS_URL and BACKUP_URL if I'm not mistaken.

Reply:
I list basically each change in the release notes.

But a label named "major change" or similar name
would be helpful to not forget to explicitly describe
such things in the section
```
New features, bigger enhancements, and possibly backward incompatible changes
```
in the release notes.

Let's further discuss it next week.

I wish you a relaxed and recovering weekend!